app/assets/stylesheets/application.css

```diff
@@ -12,6 +12,7 @@ body {
   margin: auto;
 }
 
+.wrapper,
```

Comment:
- [ Good Job ] Good job styling the `Login` and `Registration` forms.

![Screenshot from 2023-08-30 21-05-54](https://github.com/Microverse-Fullstack-Program/rails-blog-app-v2/assets/16392046/b81dec73-afeb-4f34-b2ae-730129fb405e)